disnake/guild.py

```diff
@@ -1996,6 +2006,19 @@ async def edit(
                 else:
                     features.discard("COMMUNITY")
 
+            if invites_disabled is not MISSING:
+                if community is not MISSING:
+                    raise ValueError(
```

Comment:
This should be added to the method's documentation, even if it's a temporary measure

Reply:
https://github.com/DisnakeDev/disnake/pull/718/commits/f52a3a2b328068ac17152243c652157431de284b